src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -214,11 +219,20 @@ public void writeThrough(Conversation conversation) {
   
   /** Write a Profile object to the Datastore service. */
   public void writeThrough(Profile profile) {
-    Entity conversationEntity = new Entity("chat-profiles");
+    Entity profileEntity = new Entity("chat-profiles", profile.getUserID().toString());
+    profileEntity.setProperty("user_uuid", profile.getUserID().toString());
+    profileEntity.setProperty("description", profile.getDescription());
+    
+    datastore.put(profileEntity);
+  }
+  
+  /** Update a Profile object in the DataStore service **/
+  public void updateProfile(Profile profile, String description) throws EntityNotFoundException {
+    Key key = KeyFactory.createKey("chat-profiles", profile.getUserID().toString());
```

Comment:
What is this key for?

Reply:
The Key  used is to grab a specific Profile entity from the Database. In this case I am grabbing the profile entity through the User ID.

Reply:
Oh I see. Thanks for clarifying